src/odemis/acq/orsay_milling.py

```diff
@@ -291,3 +290,20 @@ def on_procedure_info(self, param, attr):
             logging.debug('New active procedure (%s) : %s', param.Actual, self.miller.ActiveProcedureName.Target)
             self.miller.MillingActivationState.Target = 1  # Start milling once the new procedure has been loaded
             param.Unsubscribe(self.on_procedure_info)
+
+
+class FakeOrsayMilling(OrsayMilling):
```

Comment:
I could not figure out where the class is called. Where is it used?

Reply:
> It is needed when we want to test without hardware and fake the milling process. So I think it is still worthy to keep it there commented out.

That comment from the previous conversation resolves this one too I guess :)